---
layout: single
title: Research Topics
permalink: /research/
---

### Large-scale biogeography ~ Genomics

![image-right](/assets/images/Research/Slide1.png){: .align-right}
As the most abundant and diverse organisms on Earth, microbes' distribution in nature remain largely undocumented, particularly distribution at a large spatial scale. To enrich microbial databases and to improve understanding of biological strategies that microbes adopt to occupy niches, we systematically sampled soil from natural environments to collect _Listeria_ across the contiguous US. _Listeria_ is a diverse Gram-positive bacterial genus, which includes _L. monocytogenes_, the major causative agent of food-borne disease listeriosis. By mining genomic and environmental data, I am depicting the nationwide atlas of _Listeria_ and exploring mechanisms underlying the niche breadth of _Listeria_ populations. 
{: .text-justify}

Check out this fun video showing how we obtained and processed soil samples! Shout out to our amazing sampling team! Thanks to our effort in sampling at such a large scale, a number of new (and weird) _Listeria_ species have been discovered. 
{: .text-justify}

<video style="width:100%;" controls poster="/assets/images/Listeria_sampling_cover.jpg">
  <source src="/assets/videos/Listeria_sampling.webm" type="video/webm">
  <source src="/assets/videos/Listeria_sampling.mp4" type="video/mp4">
Your browser does not support displaying video. Please <a href="/assets/videos/Listeria_sampling.mp4">download</a>.
</video>

### Population structure ~ Environment 

Genetic variation in a bacterial population can occur as a result of dissimilarities of ecological niches, which may represent distinct habitats, hosts, and geographic locations. To identify environment-associated population structure and evolutionary drivers of genetic diversity, I study two bacterial genera vital to public health, _Listeria_ and _Salmonella_, using tools of population genetics and genomics.
{: .text-justify} 


![image-right](/assets/images/Research/Slide2.png){: .align-right}

*  {: .text-justify} While _Listeria_ primarily inhabits in natural environments, it has been frequently isolated from food-associated environments, which pose thread to food safety. To identify evidence of ecological differentiation and inter-environment transmission, I am characterizing _Listeria_ isolates collected from natural environment (soil) and food-associated environments (food processing facilities and agricultural water) using whole genome sequencing and comparing genomic features between isolates from the two types of environments using differential analyses. 


![image-right](/assets/images/Research/Slide3.png){: .align-right}

*  {: .text-justify} Sequencing of _sigB_, which encodes a stress responsive alternative sigma factor, has emerged as a widely used molecular tool for rapid classification of diverse _Listeria_ isolates. To assess the validity of _sigB_ allelic typing for _Listeria_, we characterized the evolutionary patterns of _sigB_ and found that this gene is a relatively stable gene free from frequent homologous recombination and positive selection. _sigB_ allelic typing is a useful tool for _Listeria_ species classification. 

   __Liao J__, Wiedmann M, Kovac J. [Genetic stability and evolution of the _sigB_ allele, used for _Listeria sensu stricto_ subtyping and phylogenetic inference.](http://aem.asm.org/content/83/12/e00306-17.abstract){:target="_blank"} _Appl Environ Microbiol_ 83:e00306-17

![image-right](/assets/images/Research/Slide4.png){: .align-right}

*  {: .text-justify} _S. enterica_, an important human and animal pathogen, contains >2,500 serotypes, which display different epidemiological and ecological characteristics. The emergence of antimicrobial-resistant (AMR) strains of _S. enterica_ poses a growing threat to public health. I carried out two studies assessing the micro-evolution of AMR _S. enterica_: i) evolution associated with host range, in which we found evidence that recent population bottleneck and genome decay are linked to the narrow host range of _S_. Dublin (a serotype adapted to bovine); and ii) population structure associated with host species and geographic origins, in which we showed that presence/absence profiles of genes (e.g., transposase and virulence genes) of AMR _S. enterica_ significantly vary by host species, while geographic origin appears to be more important in effecting its genetic diversity at single nucleotide polymorphism level.

   __Liao J__, Orsi RH, Carroll LM, Kovac J, Ou H, Zhang H, Wiedmann M. [Serotype-specific evolutionary patterns of antimicrobial-resistant _Salmonella enterica_.](https://bmcevolbiol.biomedcentral.com/articles/10.1186/s12862-019-1457-5){:target="_blank"} _BMC Evol Biol_ 19:132\\
   __Liao J__, Orsi RH, Carroll LM, Wiedmann M. [Comparative genomics reveals different population structures associated with host and geographic origin in antimicrobial-resistant _Salmonella enterica_.](https://sfamjournals.onlinelibrary.wiley.com/doi/abs/10.1111/1462-2920.15014?af=R){:target="_blank"} _Environ Microbiol_ 22:2811-2828

### Distribution and diversity ~ Ecological forces

Understanding ecological forces driving the change in microbial composition from local to regional scales is a central theme in microbial ecology and biogeography. It is known that microbial distribution and diversity can be governed by deterministic and neutral ecological processes jointly. My goal is to disentangle the roles of ecological processes underlying the distribution and diversity of microbes from specific environments and to seek generic principles. I study microbial populations as well as microbial communities with a focus on environmental selection, dispersal, and ecological drift.
{: .text-justify} 

![image-right](/assets/images/Research/Slide5.png){: .align-right}

*  {: .text-justify} High quality habitat (e.g., forest) for wildlife in watershed adjacent to produce fields may increase risk for contamination of agricultural products by enteric bacteria, including _E. coli_. Understanding how landscape of watershed impacts the distribution of edaphic _E. coli_ is thus essential in this aspect. By sampling and characterizing _E. coli_ from watersheds dominated by either forest or produce fields and by developing wildlife-driven dispersal models, we found that watershed landscape patterns influence the distribution of _E. coli_ by adjusting the importance of environmental selection and dispersal. 

   __Liao J__, Bergholz P, Wiedmann M. [Terrestrial landscape impacts the biogeographic pattern of soil _Escherichia coli_ via altering the strength of environmental selection and dispersal.](https://www.biorxiv.org/content/10.1101/2020.06.30.181495v1.abstract){:target="_blank"} _bioRxiv_ doi:10.1101/2020.06.30.181495

![image-right](/assets/images/Research/Slide6.png){: .align-right}

*  {: .text-justify} How mechanisms underlying the distribution of microbial groups with different traits is yet to be fully understood. To this end, I conducted a series of studies quantifying deterministic and neutral effects on the community assembly of bacteria sampled from high-altitude freshwater lakes in China. We found that i) habitat generalists are mainly driven by neutral processes; ii) habitat specialists are only governed by deterministic processes, with salinity identified as the top driver; iii) both abundant and rare taxa were strongly influenced by deterministic processes, but the key environmental variables differed between the two groups; and iv) _Cyanobacteria_, a group of bacteria critical for sustaining the productivity of ecosystems, is assembled via deterministic processes driven by water property, lake morphology and watershed land-use. 

   __Liao J__, Cao X, Zhao L, Wang J, Gao Z, Wang MC, Huang Y. [The importance of neutral and niche processes for bacterial community assembly differs between habitat generalists and specialists.](http://dx.doi.org/10.1093/femsec/fiw174){:target="_blank"} _FEMS Microbiol Ecol_ 92:fiw174\\
   __Liao J__, Cao X, Wang J, Zhao L, Sun J, Jiang D, Huang Y. [Similar community assembly mechanisms underlie similar biogeography of rare and abundant bacteria in lakes on Yungui Plateau, China.](http://onlinelibrary.wiley.com/doi/10.1002/lno.10455/full){:target="_blank"} _Limnol Oceanogr_ 62:723-735\\
   __Liao J__, Zhao L, Cao X, Sun J, Gao Z, Wang J, Jiang D, Fan H, Huang Y. [_Cyanobacteria_ in lakes on Yungui Plateau, China are assembled via niche processes driven by water physicochemical property, lake morphology and watershed land-use.](http://dx.doi.org/10.1038/srep36357){:target="_blank"} _Sci Rep_ 6:36357 

![image-right](/assets/images/Research/Slide7.png){: .align-right}

*  {: .text-justify} While crude oil extraction is of immense importance for energy and economy, hydrocarbons in the oil can be extremely toxic to living organisms and can interfere with the balance and function of ecosystems. Apparently, oil can also change the structure and function of microbial community in soil. By sampling and characterizing soil microbial communities from four major oil fields in China, we identified the direct and indirect impacts of oil contamination on the distribution of microbial communities, and found oil contamination lead to a drastic increase in hydrocarbon-degrading bacteria and a reduction of nitrogen-transforming bacteria in soils regardless of soil types.

   __Liao J__, Wang J, Huang Y. [Bacterial community features are shaped by geographic location, physicochemical properties and oil contamination of soil in main oil fields of China.](http://dx.doi.org/10.1007/s00248-015-0572-0){:target="_blank"} _Microb Ecol_ 70:380-389\\
   __Liao J__, Wang J, Jiang D, Wang M, Huang Y. [Long-term oil contamination causes similar changes in microbial communities of two distinct soils.s](http://dx.doi.org/10.1007/s00253-015-6880-y){:target="_blank"} _Appl Microbiol Biotechnol_ 99:10299-10310

More on my postdoc projects coming soon!

<style type="text/css">
	a {
		text-decoration: none;
	}
	body {
		font-size: 90%;
	}
</style>
